src/webview_provider/queryResultPanel.ts

```diff
@@ -135,36 +162,65 @@ export class QueryResultPanel extends AltimateWebviewProvider {
     this._disposables.push(
       workspace.onDidChangeConfiguration(
         (e) => {
-          if (!e.affectsConfiguration("dbt.enableNewQueryPanel")) {
-            return;
-          }
-          if (this._panel) {
-            this.renderWebviewView(this._panel.webview);
+          if (e.affectsConfiguration("dbt.enableQueryBookmarks")) {
+            this.updateEnableBookmarksInContext();
+            if (this._panel) {
+              this.renderWebviewView(this._panel.webview);
+            }
           }
         },
         this,
         this._disposables,
       ),
     );
-    window.onDidChangeActiveColorTheme(
-      (e) => {
-        if (this._panel) {
```

Comment:
does light/dark mode work without this?

Reply:
yes, vscode adds vscode-dark/light class name to body tag when loading webview. we are using that in webview_panels for all pages. So this will not be needed anymore